web/svg/src/jsMain/kotlin/org/jetbrains/compose/web/svg/svgAttrs.kt

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2020-2022 JetBrains s.r.o. and respective authors and developers.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package org.jetbrains.compose.web.svg
+
+import org.jetbrains.compose.web.attributes.*
+import org.jetbrains.compose.web.css.*
+import org.w3c.dom.svg.*
+
+fun AttrsScope<SVGElement>.width(px: Number) {
+    attr("width", px.toString())
+}
+
+fun AttrsScope<SVGElement>.height(px: Number) {
+    attr("height", px.toString())
+}
+
+fun AttrsScope<SVGElement>.width(value: CSSLengthOrPercentageValue) {
+    attr("width", value.toString())
+}
+
+fun AttrsScope<SVGElement>.height(value: CSSLengthOrPercentageValue) {
+    attr("height", value.toString())
+}
+
+fun AttrsScope<SVGElement>.xmlns(nameSpace: String) {
+    attr("xmlns", nameSpace)
+}
+
+fun AttrsScope<SVGElement>.attributeName(attributeName: String) {
+    attr("attributeName", attributeName)
+}
+
+fun AttrsScope<SVGElement>.fill(color: String) {
+    attr("fill", color)
+}
+
+fun AttrsScope<SVGElement>.fillRule(fill: String) {
+    attr("fill-rule", fill)
+}
+
+fun AttrsScope<SVGElement>.href(href: String) {
+    attr("href", href)
+}
+
+fun AttrsScope<SVGElement>.viewBox(viewBox: String) {
+    attr("viewBox", viewBox)
+}
+
+fun AttrsScope<SVGElement>.transform(transform: String) {
+    attr("transform", transform)
+}
+
+fun AttrsScope<SVGElement>.d(d: String) {
+    attr("d", d)
+}
+
+fun AttrsScope<SVGElement>.points(points: String) {
```

Comment:
perhaps  adding one more overload could be useful?

```kotlin
fun AttrsScope<SVGElement>.points(vararg points: Number) {
    attr("points", points.toList().chunked(2).joinToString(" ") { it.joinToString(",") })
}
```

Reply:
Sure, will add it